bin/Main.ml

```diff
@@ -60,7 +60,8 @@ let main img1Path img2Path diffPath threshold outputDiffMask failOnLayoutChange
       ->
         { exitCode = 0; diff = Some diffOutput }
     | Pixel (diffOutput, diffCount, diffPercentage, _) ->
-        IO1.saveImage diffOutput diffPath;
+        if Option.is_some diffPath then
+          IO1.saveImage diffOutput @@ Option.get diffPath;
```

Comment:
I would change this to be a `match` instead. 

```suggestion
        match diffPath with
          | None -> ()
          | Some path -> IO1.saveImage diffOutput path;
```

or even just an Option.iter:

```suggestion
        diffPath |> Option.iter (IO1.saveImage diffOutput);
```

I am on my phone currently. So its not tested and formatting might be wrong.

Reply:
I updated the code to use `Option.iter`.